src/OrchardCore/OrchardCore.Logging.NLog/HostBuilderExtensions.cs

```diff
@@ -16,7 +16,9 @@ public static IHostBuilder UseNLogHost(this IHostBuilder builder)
             .ConfigureAppConfiguration((context, _) =>
             {
                 var environment = context.HostingEnvironment;
-                LogManager.Configuration.Variables["configDir"] = environment.ContentRootPath;
+                var appData = System.Environment.GetEnvironmentVariable("ORCHARD_APP_DATA");
+                var configDir = string.IsNullOrWhiteSpace(appData) ? $"{environment.ContentRootPath}/App_Data" : appData;
```

Comment:
probably better to use `ShellOptionsSetup.DefaultAppDataPath` instead of hard coding `App_Data`

Reply:
can't use it it's private ShellOptionsSetup.DefaultAppDataPath

Reply:
Moved private constants to `ShellOptionConstants` in abstractions